client/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -73,16 +76,17 @@ const LoginForm: React.FC<Props> = props => {
       />
       {errors.password && <span className="login__field-error">{i18n.t(errors.password)}</span>}
 
-      {invitedUser?.status !== 'active' && (
+      {invitedUser && invitedUser.status !== 'active' && (
```

Comment:
this is the same as before
`invitedUser?.status !== 'active' && (`

Reply:
invitedUser?.status can return undefined and undefined !=='active' is true